locales/en.yaml

```diff
@@ -0,0 +1,58 @@
+# Generic UI components
+components:
+  spinny:
+    label:
+      ACTIVE: Active loading indicator
+      ERROR: Error indicator
+      SUCCESS: Success indicator
+      WARNING: Warning indicator
+
+  wikitext:
+    RENDERING_INDICATOR: Rendering...
+    perf:
+      TITLE: Performance
+      RENDER: "RENDER:"
+
+# Sheaf editor
+sheaf:
+  TITLE: SHEAF
+  VERSION: Alpha
+
+  tooltips:
+    CLOSE_PREVIEW: Close Preview
+    OPEN_PREVIEW: Open Preview
+
+  stats:
+    CHARS: CHARS
+    BYTES: BYTES
+    WORDS: WORDS
+    LINES: LINES
+
+  preview_tabs:
+    RESULT: Result
+    HTML: HTML Output
+    CSS: CSS Output
+    AST: AST
+
+# CodeMirror FTML language extension
+cmftml:
+  lint:
+    # Warning emit description
+    WARNING_SOURCE: >
+      ftml({rule}: {kind} at {token}) [{from, number, :: ,_}, {to, number, :: ,_}]
+    # Warnings
+    RECURSION_DEPTH_EXCEEDED: Too much recursion in markup.
+    NOT_IMPLEMENTED: The syntax ''{slice}'' hasn't been implemented in FTML yet.
+    INVALID_INCLUDE: This include is invalid and won't be rendered.
+    LIST_DEPTH_EXCEEDED: This list is nested too deeply, and can't be rendered.
+    BLOCKQUOTE_DEPTH_EXCEEDED: This blockquote is nested too deeply, and can't be rendered.
+    NO_SUCH_BLOCK: Unknown block ''{slice}''.
+    INVALID_SPECIAL_BLOCK: Block ''{slice}'' doesn't have a special invocation. (starting '*' character)
```

Comment:
In [WJ-523](https://scuttle.atlassian.net/browse/WJ-523) (#259) "special" was renamed to "star" and "modifier" was renamed to "score".